content/blog/p3-some-assembly-required/index.md

```diff
@@ -0,0 +1,327 @@
+    * Pulumi Crossguard policy packs that capture your company policies
+
+* **Developer Experience**:
+    * A set of reusable multi-language components for cross-cutting concerns/common services
+    * A set of organization templates that match your common use cases
+
+Let’s go through each of those and briefly discuss what it looks like to set that up. 
+
+## Authentication and identity management
+
+We highly recommend using GitHub for code management. So much so that we have deeply integrated GitHub into Pulumi Cloud across a number of features. While we support alternatives, this will be the easiest and more feature-rich way to configure your platform.
+
+In Pulumi Cloud, you have the ability to create organizations. A [Pulumi Cloud organization](https://www.pulumi.com/docs/pulumi-cloud/organizations/) can help you manage teams, roles, stacks, settings, and provide a dashboard across the entire organization. Pulumi Cloud also allows you to use a variety of identity providers to login, including GitHub.
+
+For simplicity’s sake, we suggest that you start with your GitHub organization. [Create the GitHub organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch), [set up teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), and [add members](https://docs.github.com/en/organizations/organizing-members-into-teams/adding-organization-members-to-a-team) to those teams, assigning either admin or user [roles](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/using-organization-roles) to each member. 
+
+Next, in Pulumi Cloud, create an organization ***with exactly the same name*** as your GitHub organization, and choose GitHub as your identity provider. When a Pulumi organization is backed by a GitHub organization, then only members of that GitHub organization may be added to the Pulumi organization. Similarly, as soon as someone loses access to the GitHub organization, they will no longer have access to the Pulumi organization. You will also be able to [import your GitHub teams](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/#github-based-teams) directly into Pulumi Cloud. Then assign your users to the same roles in Pulumi Cloud teams as they have in the associated GitHub teams.
+
+{{< figure src="teams-gh-pulumi.png" caption="Figure: Mapping GitHub orgs, teams, and roles to Pulumi">}}
+
+Finally, you can [map teams to stacks](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/#granting-access-to-stacks-within-teams) to grant access at specific permission levels. If you’re not familiar with [Pulumi Stacks](https://www.pulumi.com/docs/concepts/stack/), a stack is a materialized instance of a specific set of cloud resources, as defined in a Pulumi program.
+
+## Pulumi ESC: Managing credentials, configuration, and other secrets
```

Comment:
Should this next section be the same as the bullets above? Secrets, Configuration, and Policy:

Reply:
Problem with that is that there are three categories in that above list, but five separate sections within the product to talk about.. so it doesn't quite map 1:1. For that reason I chose to title them differently, so there wasn't an expectation of isomorphism between the two lists. Trying to gently shift the reader from the "high level, non-Pulumi specific things to think about for building an IDP" to "the specific Pulumi product features you need to use and how those work".